test/src/test/java/jenkins/agents/TLSCustomCertificateTest.java

```diff
@@ -0,0 +1,43 @@
+package jenkins.agents;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.InboundAgentRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+public class TLSCustomCertificateTest {
```

Comment:
The build seems to be not having these certificate related options,
https://github.com/jenkinsci/jenkins-test-harness/blob/42ffa3423ed97d8d589c7ea26998325019b747f3/src/main/java/org/jvnet/hudson/test/InboundAgentRule.java#L96

I see there is some argument parsing in here,
https://github.com/jenkinsci/jenkins-test-harness/blob/42ffa3423ed97d8d589c7ea26998325019b747f3/src/main/java/org/jvnet/hudson/test/InboundAgentRule.java#L523-L529

Reply:
From a five-second glance, I guess you would need to add an option to IAR to add custom agent arguments.

Reply:
thank you, got it working.